workflows/kfdrc-single-sample-genotyping-wf.cwl

```diff
@@ -5,10 +5,49 @@ label: Kids First DRC Single Sample Genotyping Workflow
-  While the Joint Genotyping Workflow is meant to be used with trios, this workflow is meant for processing single samples.
-  The key difference in this pipeline is a change in filtering between when the final VCF is gathered by GATK GatherVcfCloud and when it is annotated by VEP bcftools (see [Kids First DRC Germline SNV Annotation Workflow docs](https://github.com/kids-first/kf-annotation-tools/blob/v1.1.0/docs/GERMLINE_SNV_ANNOT_README.md) ).
-  Unlike the Joint Genotyping Workflow, a germline-oriented [GATK hard filtering process](https://gatk.broadinstitute.org/hc/en-us/articles/360035890471-Hard-filtering-germline-short-variants) is performed and CalculateGenotypePosteriors has been removed.
-  While somatic samples can be run through this workflow, be wary that the filtering process is specifically tuned for germline data.
+
+  While the Joint Genotyping Workflow is meant to be used with whole genome
+  sequenced trios, this workflow is meant for processing single samples from any
+  sequencing experiment. The key difference between the different approaches is
+  the filtering process.
+
+  While somatic samples can be run through this workflow, be wary that the
+  filtering process is specifically tuned for germline data.
+
+  ## GATK Genotype Site-Level Filtering
+
+  Coming out of the GATK Genotyping process, site-level filtering must be done to
+  remove variants that might adversely affect downstream analysis.
+
+  GATK provides many different approaches to filtering:
+  - Variant Quality Score Recalibration (VQSR)
+  - CNNScoreVariants/NVScoreVariants
+  - Variant Extract-Train-Score (VETS)
+  - Hard Filtering
+
+  The first three are all complex model-based approaches that attempt to infer
+  cutoff points based on the data provided. Hard Filtering is manually setting
+  thresholds and removing variants that fail to meet those thresholds. For this
+  workflow, we only make use of VQSR and Hard Filtering at this time.
+
+  VQSR, being a model based approach, needs sufficient data to construct that
+  model. Normally in the joint filtering context, this means having hundreds of
+  samples. According to the documentation: "it is not suitable for some
+  small-scale experiments, such as targeted gene panels or exome studies with
+  fewer than 30 exomes." Therefore, VQSR is only activated in this workflow when
+  the input gVCFs for this workflow come from whole genome sequencing experiments
+  or when the user provides 30 or more exome gVCFs.
+
+  Hard Filtering is really only constrained by having sufficient depth. In the
+  case of exome and targeted sequencing, the depths are more than sufficient. Our
+  current approach for hard filtering mirrors the default approach outlined in
+  the GATK documentation. However as they point out, "You absolutely SHOULD
```

Comment:
Is there link to the various filtering possibilities/syntax?

Reply:
There are but they are GATK doc links, which means they'll just end up dead sooner or later.

Reply:
I'm debating making archive links to everything, but in the ticket.